frontend/src/app/spot/styles/sass/components/drop-modal.sass

```diff
@@ -35,6 +34,9 @@
       width: calc(100vw - (2 * #{$spot-spacing-1}))
       height: calc(#{var(--app-height)} - (#{$spot-spacing-3_5} + #{$spot-spacing-1}))
 
+      &_not-full-screen
```

Comment:
We just recently decided that on mobile, the modals should be centered. For the drop-modal this was not relevant until now as they always span the whole height. With that new option, we have to take care of that. Currently the progress modal is shown at the bottom.